src/Controller/ReviewController.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Contractor;
+use App\Entity\Reservation;
+use App\Entity\Review;
+use App\Service\SerializerService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ReviewController extends AbstractController
+{
+    /**
+     * @Route("/api/contractor/{contractorKey}/get-reviews/", name="GET")
+     * @param string $contractorKey
+     * @param SerializerService $json
+     * @return JsonResponse
+     */
+    public function getReviews(string $contractorKey, SerializerService $json): JsonResponse
+    {
+        $contractor = $this->getDoctrine()->getRepository(Contractor::class)
+                        ->findOneBy(
+                            [
+                                'verificationKey' => $contractorKey
+                            ]
+                        );
+        if ($contractor === null) {
```

Comment:
```suggestion
        if ($contractor === null || $reviews === null) {
```
And remove else from following condition